server/src/test/scala/org/http4s/server/middleware/ResponseTimingSuite.scala

```diff
@@ -31,21 +43,24 @@ class ResponseTimingSuite extends Http4sSuite {
     val res = app(req)
 
     val header = res
-      .map(_.headers.find(_.name == CIString("X-Response-Time")))
-    header
-      .map(_.forall(_.value.toInt === artificialDelay))
-      .assertEquals(true)
+      .map(_.headers.find(_.name == CIString("X-Response-Time")).map(_.value.toInt))
+
+    header.assertEquals(Some(artificialDelay))
   }
 }
 
 object Sys {
-  private val currentTime: Ref[IO, Long] = Ref.unsafe[IO, Long](System.currentTimeMillis())
+  private val currentTime: Ref[IO, Long] = Ref.unsafe[IO, Long](0L)
```

Comment:
CE timers now return a `FiniteDuration`, so we have to call `toUnit`, which returns a `Double`. Converting that to a `Long` will sometimes lose precision and round down.. so I'm just fixing it to a value that works